client/blocks/reader-import-button/index.jsx

```diff
@@ -39,7 +39,17 @@ class ReaderImportButton extends Component {
 		}
 
 		this.fileName = file.name;
-		const req = wpcom.undocumented().importReaderFeed( file, this.onImport );
+		const req = wpcom.req.post(
+			{
+				path: '/read/following/mine/import',
+				formData: [ [ 'import', file ] ],
+			},
+			{
+				apiVersion: '1.2',
+			},
+			null,
+			this.onImport
+		);
```

Comment:
I would also copy the comment about the `query` and `body = null` being weird. One day we should fix that.

Reply:
Sure! I've added it in 2e91634cf55345d5f0950697455ec90322a95931